packages/replay-next/components/console/renderers/LogPointRenderer.tsx

```diff
@@ -148,12 +138,15 @@ function AnalyzedContent({ logPointInstance }: { logPointInstance: PointInstance
     [timeStampedHitPoint]
   );
 
-  const entry = analysisResults(timeStampedHitPoint);
-  if (entry === null) {
-    console.error(`No analysis results found for execution point "${timeStampedHitPoint.point}"`);
-
-    return <span className={styles.AnalysisError}>Analysis error</span>;
-  }
+  assert(focusRange, "No focusRange in LogPointRenderer");
```

Comment:
This _looks_ like an unsafe change in behavior. (The `range` property in `FocusContext` _can be null_.)

 We used to check and handle the case where there was no focus region.

What changed (outside of this code) that makes that check no longer necessary?

Reply:
Could you add an inline comment to this effect? That relationship is kind of distance and might not be apparent.